ie8/index.html

```diff
@@ -7,7 +7,7 @@
     <title>React Bootstrap IE8 test page</title>
 
     <!-- Bootstrap -->
-    <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css" rel="stylesheet">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Nope,  opening it locally chrome will assume the protocol is `file:` instead of http when unspecified at least on windows. This whole section though doesn't seem to work at anyway hasn't been updated since react 10. Maybe just an old artifact